gomodtidy.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
not sure about this one

Reply:
Oh that's a script I wrote for updating go modules! My bad, I'll remove that